packages/ffe-datepicker-react/src/calendar/Calendar.js

```diff
@@ -172,7 +172,7 @@ export default class Calendar extends Component {
             <ClickableDate
                 date={date}
                 month={calendar.focusedMonth()}
-                year={calendar.focusedYear()}
+                year={calendar.focusedYear().toString()}
```

Comment:
Det burde være lov å dytte inn et `number` her, så jeg synes det burde fikses i ClickableDate.

Reply:
Jeg tenker at jo strengere typing jo bedre. Poenget med typing faller litt bort når man tillater flere forskjellige typer, og så må man potensielt deale med flere forskjellige typer på samme tid inni funksjonen sin. Om typen skal være `string` eller `number` er litt hipp som happ for min del, men jeg syns den *ikke* burde tillate `string` *og* `number`.

Reply:
Har sjekket litt i koden hva som skjer. Som jeg ser det er er ClickableDate en intern komponent og `year` er alltid en `number`. Enig?

Reply:
Enig med deg der @kwltrs